sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -72,7 +72,7 @@
   },
   "dependencies": {
     "@azure/logger": "^1.0.0",
-    "@azure/schema-registry": "1.0.2",
+    "@azure/schema-registry": "^1.0.1",
```

Comment:
was there a specific reason why this version was pinned earlier?

Reply:
because it was not the stable until recently and there were breaking changes between versions.